src/components/MultipleAvatars.js

```diff
@@ -16,62 +18,59 @@ const defaultProps = {
     optionIsFocused: false,
 };
 
-const MultipleAvatars = props => (
-    <>
-        {props.avatarImageURLs !== undefined && (
-            <>
+const MultipleAvatars = ({avatarImageURLs, optionIsFocused}) => {
+    if (!_.get(avatarImageURLs, 'length')) {
```

Comment:
We tend to discourage using `lodashGet()` to access stuff like array properties and it is more common to see a default to an empty array first and then

```js
if (!avatarImagesURLs.length) {
```

Reply:
sounds good, I'll remove the get call. don't remember exactly why I added this check but maybe late-night-me just wanted a sanity check to not crash if a primitive type was somehow passed in as the prop